src/CookiePhysicsComponent.h

```diff
@@ -3,6 +3,13 @@
 
 #include "PhysicsComponent.h"
 
+typedef struct {
+    int hits;
+    double launchTime;
+    glm::vec3 launchPosition;
+    glm::vec3 hitPosition;
```

Comment:
Yeah for sure, here probably isn't the place to figure it out. Solid start, I'm good with what is here.